views/index.html.twig

```diff
@@ -34,8 +34,8 @@
             {% if name %}{{ name }}{% else %}Composer repository{% endif %}
         </h1>
 
-        <small id="lastupdate">
-            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
+        <small>
+            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::RFC3339')) }}">{{ "now"|date(constant('\DateTime::RFC3339')) }}</time>
```

Comment:
Actually according to this, RFC3339 is the valid format: https://www.w3.org/TR/html-markup/datatypes.html#common.data.datetime